package.json

```diff
@@ -50,7 +50,6 @@
     "expo-font": "~11.0.1",
     "expo-linear-gradient": "~12.0.1",
     "expo-localization": "~14.0.0",
-    "expo-modules-core": "~1.0.3",
```

Comment:
Removing this since when you spin up a new Expo app via Expo CLI it isn't in there